package/kedro_viz/integrations/kedro/hooks.py

```diff
@@ -0,0 +1,106 @@
+# pylint: disable=broad-exception-caught
+# pylint: disable=protected-access
+"""`kedro_viz.integrations.kedro.hooks` defines hooks to add additional
+functionalities for a kedro run."""
+
+import json as json_lib
```

Comment:
Can we keep this as `json`?

Reply:
in some files we have json from Kedro datasets. To distinguish the library import from kedro datasets, I used this convention through out.

Reply:
If we have json from a dataset doesn't it make more sense to call that `json_dataset` and use `json` here since this is the actual json library we're calling?